packages/utils/src/loader/svg-loader.ts

```diff
@@ -1,21 +1,18 @@
-import { LocalCache } from '../cache.js';
 import { CDNLoader } from './cdn-loader.js';
 
-const cache = new LocalCache('svg-loader', { storage: 'indexeddb' });
-
 /**
  * Checks a string to see if it's a valid URL
  * @param str String to test
  * @returns is URL
  */
-const isUrl = (str: string): boolean => (/^(https?:\/{2}|\.?\/)/i).test(str);
+export const isUrl = (str: string): boolean => (/^(https?:\/{2}|\.?\/)/i).test(str);
```

Comment:
Since this is the SvgLoader, wouldn't it make sense to also match the svg extension at the end of the url?